scripts/pylint_extensions.py

```diff
@@ -794,6 +794,55 @@ def visit_functiondef(self, node):
             self.add_message('function-args-order-cls', node=node)
 
 
+class RestrictedImportChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks restricted imports."""
```

Comment:
The description seems to be a bit vague. Can you please make it more concrete, as in, what the checker exactly restricts?

Reply:
OK changed the description PTAL.